pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -29,9 +29,7 @@
 , mpg123
 , twolame
 , gtkSupport ? false, gtk3 ? null
-  # As of writing, jack2 incurs a Qt dependency (big!) via `ffado`.
-  # In the future we should probably split `ffado`.
```

Comment:
Is this no longer true? Given

> increases its size from 163.5 MB to 164.4 MB.

It either no longer incurs the Qt dependency, or depends on Qt already via some other path.

Do you happen to know which one it is?

Reply:
1. I'm changing `jack2` to `libjack2` (the jack client library) which does not depend on `ffado`.
2. `jack2` uses only `ffado.out` which does not depend on Qt.

Reply:
Ah great, I see!